docs/verification.md

```diff
@@ -0,0 +1,174 @@
+# Verification
+
+## Offline Verification
+
+In offline Rekor storage mode Gitsign will store a HashedRekord in Rekor
+corresponding to the commit content.
+
+Unfortunately this is a bit complex to query manually. Roughly this is:
+
+```
+sha256(der(sort(system time | commit data | content type)))
+```
+
+The resulting Rekor log entry fields and inclusion proof will be stored in the
+PKCS7 object as unauthenticated (i.e. not included in the cryptographic
+signature) attributes.
+
+```
+unauth_attr:
+  object: undefined (1.3.6.1.4.1.57264.3.8)
```

Comment:
Nit: maybe worth inlining a table describing these extensions? 

(It's a bit of a lie, but you could also annotate this data too.)

Reply:
I don't want to keep this in sync with Rekor. It's only one value now, so hopefully it's less of a burden.